prow/config.yaml

```diff
@@ -7507,7 +7507,35 @@ postsubmits:
         resources:
           requests:
             memory: "6Gi"
-
+  - name: ci-kubernetes-bazel-build-1-11
+    agent: kubernetes
+    branches:
+    - release-1.11
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180511-51cf02e75-1.10
```

Comment:
We also need to come back and fork this image to have a 1.11 variant, I'll do that later.